api/WorkerGlobalScope.json

```diff
@@ -47,56 +47,6 @@
           "deprecated": false
         }
       },
-      "console": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WorkerGlobalScope/console",
-          "support": {
-            "chrome": {
-              "version_added": "4"
-            },
-            "chrome_android": {
-              "version_added": true
-            },
-            "edge": {
-              "version_added": "12"
-            },
-            "firefox": {
-              "version_added": "29",
-              "notes": "Before Firefox 30, the console would be of type <code>WorkerConsole</code> instead of <code>Console</code>."
```

Comment:
This is not an important note, the name of the interface doesn't matter when using `console.log`. By now, `console` is a namespace, so there's not even anything one can do with an interface object which will work in all browsers.